src/collider/include/collider.h

```diff
@@ -24,13 +24,11 @@ class Collider {
  public:
   Collider() {}
   Collider(const VecDH<Box>& leafBB, const VecDH<uint32_t>& leafMorton);
-  // Aborts and returns false if transform is not axis aligned.
   bool Transform(glm::mat4x3);
   void UpdateBoxes(const VecDH<Box>& leafBB);
-  // Collisions returns a sparse result, where i is the query index and j is
-  // the leaf index where their bounding boxes overlap.
   template <typename T>
-  SparseIndices Collisions(const VecDH<T>& queriesIn) const;
+  SparseIndices Collisions(const VecDH<T>& queriesIn,
+                           bool selfCollision = false) const;
```

Comment:
I updated the Collider so that we can collide a vector with itself and not spend a bunch of time reporting that every index collides with itself. This might come in handy down the road too.